services/infra/InfraService.ts

```diff
@@ -35,15 +35,22 @@ export default class InfraService {
   }
 
   createSite = async ({
-    email,
+    requesterEmail,
     repoName,
   }: {
-    email: string
-    repoName: string
+    requesterEmail: string
+    repoName: string | undefined
```

Comment:
Is this because FormSG doesn't prevent empty strings? Could we leave a comment if this is the case?

Additionally, we could do a check for namespace collision by looking at the `repos` tables to see if any record with this `repoName` already exists. This could be addressed in a separate issue/PR.

Reply:
FormSG can prevent empty strings by making a field required, but I thought it best not to depend on that. It will always be possible to modify the form so that the field is no longer required.

> Additionally, we could do a check for namespace collision by looking at the repos tables to see if any record with this repoName already exists. 

Right! I forgot that my previous PR ensured this by adding a `unique: true` constraint to the repository name column. I'll add this back or make a note to add it later.

Reply:
I just realized that there is a `name` column in both the `sites` table and `repos` table, and they both appear to contain the same thing. What's the rationale behind storing the repo name in the sites table? Can we use `sites.name` for a human-readable name instead?

Reply:
I just discussed this with @prestonlimlianjie . He clarified that `sites.name` is meant to be visible to users, while `repos.name` is meant to contain the GitHub repository name.